src/wp-settings.php

```diff
@@ -318,8 +318,6 @@
 require ABSPATH . WPINC . '/class-wp-block-type-registry.php';
 require ABSPATH . WPINC . '/class-wp-block.php';
 require ABSPATH . WPINC . '/class-wp-block-list.php';
-require ABSPATH . WPINC . '/class-wp-block-parser-block.php';
-require ABSPATH . WPINC . '/class-wp-block-parser-frame.php';
```

Comment:
These can remain (they were reordered in the other PR to prevent the fatal by requiring `class-wp-block-parser.php` first.